packages/appkit/vitest.config.ts

```diff
@@ -1,3 +1,20 @@
 import { defineProject } from 'vitest/config'
 
-export default defineProject({})
+// Exclude the exports directory from the test suite
+export default defineProject({
+  test: {
+    // @ts-ignore
+    coverage: {
+      exclude: [
+        'exports',
+        'node_modules',
+        'dist',
+        'tests/**',
+        './*.ts',
+        './*.json',
+        './*.ts',
+        './*.md'
```

Comment:
could we move this into the root file?

Reply:
tried but wasn't able to make it work properly, can do on a diff pr